controllers/apply/awards-for-all/fields.js

```diff
@@ -1173,6 +1173,43 @@ module.exports = function fieldsFor({ locale, data = {}, flags = {} }) {
                 }
             ]
         },
+        organisationHasDifferentTradingName: {
```

Comment:
If added a new field I'd suggest using `RadioField` from the new field types. Something like:

```js
new RadioField({
    locale: locale,
    name: 'organisationHasDifferentTradingName',
    label: localise({
        en: `Does your organisation use a different name in your day-to-day work?`,
        cy: `@TODO i18n`
    }),
    options: [
        {
            value: 'yes',
            label: localise({ en: `Yes`, cy: `@TODO i18n` })
        },
        {
            value: 'no',
            label: localise({ en: `No`, cy: `@TODO i18n` })
        }
    ],
    messages: [
        {
            type: 'base',
            message: localise({
                en: 'Select an option',
                cy: 'Dewis opsiwn'
            })
        }
    ]
})
```